project/Dependencies.scala

```diff
@@ -69,5 +70,6 @@ object Dependencies {
     val snowplowRawEvent = "com.snowplowanalytics"  % "snowplow-thrift-raw-event" % V.snowplowRawEvent
     val collectorPayload = "com.snowplowanalytics"  % "collector-payload-1"       % V.collectorPayload
     val badRows          = "com.snowplowanalytics" %% "snowplow-badrows"          % V.badRows
+     val kcl = "software.amazon.kinesis" % "amazon-kinesis-client" % V.kcl
```

Comment:
:art: format

Reply:
Apologies, resolved as I assumed scalfmt would fix, but it didn't. Fix incoming.